Engine/TCP/src/tcp_client/EcsApi.cs

```diff
@@ -8,6 +8,10 @@ public class EcsApi
     public EcsApi(String host, int port) {
         _client = new MyTcpClient(host, port);
     }
+    public string get_root() {
```

Comment:
same here :3
is the return value modified? if no, then return const std::string&